src/widget/OktaSignIn.js

```diff
@@ -41,17 +41,16 @@ var OktaSignIn = (function () {
     }
 
     /**
-     * Refresh the idToken
-     * @param idToken - idToken generated from the OAUTH call
-     * @param callback - function to invoke after refreshing the idToken.
-     *        The callback will be passed a new idToken if successful and
+     * Renew the passed token
+     * @param token - token to renew
+     * @param callback - function to invoke after renewing the token.
+     *        The callback will be passed a new token if successful and
      *        an error message if not.
-     * @param opts - OAUTH options to refresh the idToken
      */
-    function refreshIdToken (idToken, callback, opts) {
-      authClient.idToken.refresh(opts).then(callback)
```

Comment:
so the `authClient.idToken` was removed in auth-js-2.x?

Reply:
Yepp! https://github.com/okta/okta-auth-js/blame/ff1dec7d5b4d3c35778ae438bbd1fce0462ede95/CHANGELOG.md#L20